wikiconv/ingest_revisions/dataflow_main.py

```diff
@@ -23,78 +24,56 @@
 python dataflow_main.py --setup_file ./setup.py
 
 Args:
-
 ingestFrom: choose from the three options : {wikipedia, local, cloud}:
-  - wikipedia: performs the downloading job from Wikipedia, run with:
-               [python dataflow_main.py --setup_file ./setup.py\
-                   --ingestFrom=wikipedia --download --language=YourLanguage\
-                   --dumpdate=YourDumpdate --cloudBucket=YourCloudBucket\
-                   --project=YourGoogleCloudProject --bucket=TemporaryFileBucket]
-  - local: Tests the pipeline locally, run the code with
-           [python dataflow_main.py --setup_file ./setup.py\
-            --ingestFrom=local --localStorage=YourLocalStorage --testmode\
-            --output=YourOutputStorage --project=YourGoogleCloudProject\
-            --bucket=TemporaryFileBucket]
+  - wikipedia: performs the downloading job from Wikipedia, run with: [python
+    dataflow_main.py --setup_file ./setup.py --ingestFrom=wikipedia --download
```

Comment:
line breaks here will make copy paste really horrible.

Reply:
but backslashes in strings cause lint complaints - will try to find a trade off.